### DISCUSSION TITLE
the weight and height of cancer population

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1386] [Questions & Problems]
Hello everyone, I am trying to use pk-sim to establish a virtual population recently, and I am looking for relevant literature to determine the demographic parameters of cancer population, but I use the relationship between weight and height in the literature, and the weight calculated according to the height is very strange. When the height is measured in centimeters, the weight is too large and unreasonable, and when measured in meters, the weight is too small. Would you like to ask if you have encountered this situation? Looking forward to your reply!
![WT-height1](https://user-images.githubusercontent.com/95362728/202329626-b17bb8f1-8bdf-48b6-bae7-db76113660b6.jpg)
![WT-height2](https://user-images.githubusercontent.com/95362728/202329645-545072f4-83e2-4c29-b50a-0592ec67fb14.jpg)
![WT-height3](https://user-images.githubusercontent.com/95362728/202329654-fd0172d1-3f2a-41a8-88b8-223b15f2762f.jpg)
![WT-height4](https://user-images.githubusercontent.com/95362728/202329662-da356fc8-ad76-4910-83eb-e288ac3cf4e2.jpg)

**Comment by prvmalik** [comment]
Cancer populations vary widely depending on the indication. I would advise using the age brackets defined by the epidemiological information in literature for the type of cancer you are interested in, and allowing height and weight to fall to default. You may look at decreasing weight (fat or muscle mass) manually thereafter if you discover that cancer patients of your indication are leaner than otherwise healthy subjects.

**Reply by WTY-tt** [reply]
Thank you for your reply. I try to look at decreasing weight (fat or muscle mass) manually thereafter.
